### PR TITLE
issue/49 Re-write for fw v4

### PR DESCRIPTION
**Pull request by oliverfoster**
#49 

### Changed
* Improved accessibility
* Moved animations into CSS
* Used flex for tabs placement
* Allowed both tabs and arrows
* Simplified article extensions
* Refactored and distilled behaviour
* Bumped fw version to v4

### Removed
* Animation duration properties as these are now done in the CSS
* `jquery.resize` library is redundant as is in the core

### Tested
* JAWS 2020, Chrome and IE11
* NVDA, Chrome and IE11

Need to test safari osx

### CODE REVIEW

File: js/view.js [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+      this.updateControlsState();
+      this.updateHeight();
+      this.$('.component').on('resize', this.updateHeight);
+    },
+
+    hideInactiveItems: function() {
+      var isEnabled = this.isEnabledOnScreenSize();
+      if (!isEnabled) {
+        var $blocks = this.$('.article-block-slider > .block-container > .block');
+        $blocks.removeClass('is-visible');
```
Comment (oliverfoster) [inline js/view.js]: should just be a return early

File: js/view.js [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+    onActiveItemChange: function(model, value) {
+      if (!value) return;
+      this.moveToActiveItem();
+    },
+
+    moveToActiveItem: function() {
+      var isEnabled = this.isEnabledOnScreenSize();
+      if (!isEnabled) return;
+      var $container = this.$('.article-block-organise');
+      // Immediately move to a focs holder to prevent the button from double reading
```
Comment (oliverfoster) [inline js/view.js]: typo `focs` > `focus`

File: js/view.js [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+    },
+
+    onTransitionEnd: function(event) {
+      var $target = $(event.currentTarget);
+      if (!$target.is('.block-container')) return;
+      var $container = this.$('.article-block-organise');
+      var currentBlock = this.model.getActiveItem().get('_index');
+      // Modify the dom to hide other items
+      this.hideInactiveItems();
+      // Shift focus to the next item after the animation as jaws struggle with off-screen content
```
Comment (oliverfoster) [inline js/view.js]: typo `struggle` to `struggles`

**Review by guywillis — APPROVED** [review]
👍

**Pull request merged**